### DISCUSSION TITLE
Where are the log files?

### DISCUSSION THREAD

**Discussion by func0der** [vtsykun/packeton#94] [Q&A]
Hey,

I am trying to proxy a repository. It is not packagist.org, it is a nexus composer group which includes a proxy of packagist.org and additional private packages. I do not think that this is important, though.

I added the proxy in the config and ran `./bin/console  packagist:sync:mirrors proxyname`

Then when I put the proxy in my composer.json and try to download packages, I get errors like this:

```
..../p2/monolog/monolog.json" file could not be downloaded (HTTP/1.1 500 Internal Server Error):
```

I can not find any logs about this though. Where do the application logs go?

**Comment by vtsykun** [comment]
Hi, app dir `/var/log/prod.log`

**Reply by func0der** [reply]
There is no log file :thinking: 

Maybe I need to clarify my setup more.

I downloaded the repo and just ran `docker compose up`. It is probably not the `prod`  setup.

**Reply by vtsykun** [reply]
Maybe write permission issue `appdir/var` to this dir.

Also you can try enable dev debug to see details errors  
1) Change docker env var 

```
services:
..
        environment:
            APP_ENV: dev
```
2) Login to container and run `composer install` to install all dev dependencies
